browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -276,22 +280,25 @@ class RewardsDOMHandler
       const brave_rewards::RewardsNotificationService::RewardsNotificationsList&
           notifications_list) override;
 
-  // AdsServiceObserver implementation
-  void OnAdRewardsDidChange() override;
-  void OnNeedsBrowserUpgradeToServeAds() override;
+  // bat_ads::mojom::BatAdsObserver implementation
+  void OnBraveRewardsDidChange() override;
```

Comment:
What event triggers this handler? Can we use a name that is a little more self-describing?

Reply:
This is triggered after successfully initializing ads on browser launch and when the statement of accounts changes for brave://rewards.

Reply:
Fixed